src/main/java/reposense/RepoSense.java

```diff
@@ -136,6 +139,10 @@ public static void main(String[] args) {
             }
         }
 
+        if (configs.isEmpty()) {
+            throw new InvalidLocationException("All locations are invalid. Please supply at least one valid location.");
```

Comment:
I think we can change the message to just `All repository locations are invalid`.
`repository locations` is a little clearer than `locations`.
We can omit the sentence `Please supply at least one valid location.` as it's self explanatory.

Reply:
Sure! I'll make the changes right now. Then do I need to do anything to pass the `codecov` check? cos I'm a bit confused with it

Reply:
> Then do I need to do anything to pass the codecov check?

No further work needed for this.
In general, extremely small changes to code coverage may not be meaningful.

Reply:
Sure! Thank you for clarification!